app/api/services/informationextraction/InformationExtraction.ts

```diff
@@ -119,7 +119,7 @@ class InformationExtraction {
           data = {
             ...data,
             language_iso: languages.get(file.language!, 'ISO639_1') || defaultTrainingLanguage,
-            label_text: propertyLabeledData.selection?.text,
+            label_text: file.label_text,
```

Comment:
Minor semantic thing: could we name label_text something else, which shows that this is the value from the entity? entity_label_text, entity_value_text, etc.